Source/Core/Core/TitleDatabase.cpp

```diff
@@ -56,6 +56,9 @@ void TitleDatabase::AddLazyMap(DiscIO::Language language, const std::string& lan
   m_title_maps[language] = [language_code]() -> Map {
     return LoadMap(File::GetSysDirectory() + "wiitdb-" + language_code + ".txt");
   };
+  m_trititle_maps[language] = [language_code]() -> Map {
+    return LoadMap(File::GetSysDirectory() + "tritdb-" + language_code + ".txt");
```

Comment:
Any particular reason why Triforce has been shortened to "tri" in "tritdb" and "tri_id"? Normally we don't use these kinds of abbreviations. Especially the file name might be difficult to change later.

Reply:
At the time I think I did it because it lined up neatly with `wiitdb`, but I'm happy to change it if you'd prefer the non-abbreviated version.

Reply:
Yes, I would prefer it non-abbreviated.

Reply:
Fixed.